packages/transform-to-vanilla/src/transform.ts

```diff
@@ -1,17 +1,105 @@
 import type { ComplexStyleRule } from "@vanilla-extract/css";
-import type { ComplexCSSRule, CSSRule } from "./types/style-rule";
+import type {
+  ComplexCSSRule,
+  CSSRule,
+  VanillaStyleArray,
+  VanillaClassNames
+} from "./types/style-rule";
 
-// TODO: Need traverse
+// == Interface ================================================================
 export function transform(style: ComplexCSSRule): ComplexStyleRule {
   if (Array.isArray(style)) {
     return style.map((eachStyle) => {
-      return eachStyle;
+      return isClassNames(eachStyle) ? eachStyle : transformStyle(eachStyle);
     });
   }
   return transformStyle(style);
 }
 
+// == Utils ====================================================================
+function isClassNames(
+  style: VanillaStyleArray[number]
+): style is VanillaClassNames {
+  return typeof style === "string" || Array.isArray(style);
+}
+
 // TODO: Need traverse
 function transformStyle(style: CSSRule) {
   return style;
 }
```

Comment:
If this todo will be done, test code is also gonna be changed?

Reply:
It is separated into different files #6

Reply:
thx i will check!